lib/node_modules/@stdlib/math/base/special/gcdf/src/main.c

```diff
@@ -0,0 +1,183 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/gcdf.h"
+#include "stdlib/math/base/special/fmodf.h"
+#include "stdlib/math/base/assert/is_nanf.h"
+#include "stdlib/math/base/assert/is_integerf.h"
+#include "stdlib/constants/float32/pinf.h"
+#include "stdlib/constants/float32/ninf.h"
+#include <stdint.h>
+
+// 2^32 - 1
+static const int32_t STDLIB_CONSTANT_INT32_MAX = 2147483647;
```

Comment:
The comment is wrong (it's `2^31 - 1`), but we can also just use the stdlib constant:

```suggestion
#include "stdlib/constants/float32/ninf.h"
#include "stdlib/constants/int32/max.h"
#include <stdint.h>
```